tools/reactive-controllers/src/delegate-controllers/SpectrumTwoDownStateController.ts

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Please don't call this `SpectrumTwo`. It only happens to be a `DownStateController` that applies to "Spectrum Two".

Reply:
Renamed to `DownStateController`